frontend/src/metabase/collections/containers/CollectionContent.jsx

```diff
@@ -284,9 +290,30 @@ export default class CollectionContent extends React.Component {
             </PinDropTarget>
           )}
           <Box className="relative" mt={1}>
-            {// if there are pins and we also have items show a heading for this section
-            collectionHasPins && unpinnedItems.length > 0 && (
-              <CollectionSectionHeading>{t`Everything else`}</CollectionSectionHeading>
+            {showFilters ? (
+              <ItemTypeFilterBar
+                analyticsContext={ANALYTICS_CONTEXT}
+                filters={ITEM_TYPE_FILTERS.map((f, i) => {
```

Comment:
Maybe `assocIn` is more concise/readable?
```js
const everythingName= collectionHasPins && unpinned.length > 0 ? t`Everything else` : t`Everything`;
const filters = assocIn(ITEM_TYPE_FILTERS, [0, 'name'], everythingName)
```

Reply:
Yeah it sure is!